test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -71,25 +139,58 @@ def _test_load_tx_ethbridge_burn(ctx: test_utils.EnvCtx, amount_per_tx: int, tra
     sif_balances_initial: List[cosmos.Balance] = [ctx.get_sifchain_balance(sif_acct) for sif_acct in sif_accts]
     eth_balances_initial: List[str] = [ctx.eth.get_eth_balance(eth_acct) for eth_acct in eth_accts]
     assert all([b == 0 for b in eth_balances_initial])  # Might be non-zero if we're recycling accounts
+    
+    if token_address != None:
+        erc20_balances_initial: List[str] = [ctx.get_erc20_token_balance(token_address, eth_acct) for eth_acct in eth_accts]
+        assert all([b == 0 for b in eth_balances_initial])  # Might be non-zero if we're recycling accounts
+
 
     # Create a dispensation sif account that will receive all locked ETH and dispense it to each sif account
     # (we do this in one transaction because lock transactions take a lot of time).
     # Dispensation account needs rowan for distributing ceth to sif_accts.
-    dispensation_sif_acct: cosmos.Address = ctx.create_sifchain_addr(fund_amounts={rowan: n_sif * sif_tx_fee_in_rowan})
+    if token_denom == rowan:
+        amount = sum_all * amount_per_tx + n_sif * 2 * sif_tx_fee_in_rowan
+        dispensation_sif_acct: cosmos.Address = ctx.create_sifchain_addr(fund_amounts={rowan: amount})
+    elif token_denom == ctx.ceth_symbol:
+        dispensation_sif_acct: cosmos.Address = ctx.create_sifchain_addr(fund_amounts={rowan: n_sif * sif_tx_fee_in_rowan})
+    else:
+        # for erc20 token, dispensation_sif_acct need distribute both ceth for cross chain fee and erc20 to burn
+        dispensation_sif_acct: cosmos.Address = ctx.create_sifchain_addr(fund_amounts={rowan: n_sif * 2 * sif_tx_fee_in_rowan})
 
     # Transfer ETH from operator to dispensation_sif_acct (aka lock)
     old_balances = ctx.get_sifchain_balance(dispensation_sif_acct)
-    ctx.bridge_bank_lock_eth(ctx.operator, dispensation_sif_acct, sum_all * (amount_per_tx + sif_tx_burn_fee_in_ceth))
+    lock_eth_amount = sif_tx_burn_fee_in_ceth
+    if token_denom == ctx.ceth_symbol:
+        lock_eth_amount += amount_per_tx
+    ctx.bridge_bank_lock_eth(ctx.operator, dispensation_sif_acct, sum_all * lock_eth_amount)
     ctx.advance_blocks()
-    new_balances = ctx.wait_for_sif_balance_change(dispensation_sif_acct, old_balances)
+    old_balances = ctx.wait_for_sif_balance_change(dispensation_sif_acct, old_balances)
+
+    # just for erc20 token
+    if token_address != None and not isRowan:
```

Comment:
While `token_address != None` will work, the correct (idiomatic) way is to use `token_address is not None`.